evalml/tests/automl_tests/test_iterative_algorithm.py

```diff
@@ -253,6 +285,38 @@ class MockTextClassificationPipeline(BinaryClassificationPipeline):
     assert pipeline[0]._all_text_columns == ['text_col_1', 'text_col_2']
 
 
+@pytest.mark.parametrize("n_jobs", [-1, 0, 1, 2, 3])
+def test_iterative_algorithm_stacked_ensemble_n_jobs_binary(n_jobs, dummy_binary_pipeline_classes):
+    dummy_binary_pipeline_classes = dummy_binary_pipeline_classes()
+    algo = IterativeAlgorithm(allowed_pipelines=dummy_binary_pipeline_classes, ensembling=True, n_jobs=n_jobs)
+    next_batch = algo.next_batch()
+
+    scores = range(0, len(next_batch))
+    for score, pipeline in zip(scores, next_batch):
+        algo.add_result(score, pipeline)
+    for i in range(5):
+        next_batch = algo.next_batch()
+        for pipeline in next_batch:
+            if isinstance(pipeline.estimator, StackedEnsembleClassifier):
+                assert pipeline.parameters['Stacked Ensemble Classifier']['n_jobs'] == n_jobs
+
+
+@pytest.mark.parametrize("n_jobs", [-1, 0, 1, 2, 3])
+def test_iterative_algorithm_stacked_ensemble_n_jobs_regression(n_jobs, dummy_regression_pipeline_classes):
+    dummy_regression_pipeline_classes = dummy_regression_pipeline_classes()
+    algo = IterativeAlgorithm(allowed_pipelines=dummy_regression_pipeline_classes, ensembling=True, n_jobs=n_jobs)
+    next_batch = algo.next_batch()
+
+    scores = range(0, len(next_batch))
+    for score, pipeline in zip(scores, next_batch):
+        algo.add_result(score, pipeline)
+    for i in range(5):
+        next_batch = algo.next_batch()
+        for pipeline in next_batch:
+            if isinstance(pipeline.estimator, StackedEnsembleRegressor):
```

Comment:
For both tests, can we add a check that there *is* an ensemble method? I'm not sure 5 batches is enough, and this test will pass if there is no ensemble to check for.